ui/src/tool-panel/km-post/dialog/km-post-edit-store.ts

```diff
@@ -54,13 +54,64 @@ export const initialKmPostEditState: KmPostEditState = {
     allFieldsCommitted: false,
 };
 
+type GkBounds = { min: Point; max: Point };
+
 export const WGS_84_PROJECTION = '+proj=longlat +ellps=WGS84 +datum=WGS84 +no_defs';
 
-const SOUTHERNMOST_POINT_OF_FINLAND_DEG = 59.505;
-const NORTHMOST_POINT_OF_FINLAND_DEG = 70.092;
-const NORTHING_MARGIN_DEG = 0.1;
 const EASTING_MARGIN_BETWEEN_GKS_DEG = 0.01;
 
+const GK_BOUNDS: GkBounds[] = [
```

Comment:
Ruma kovakoodattu taulukko mutta minkäs teet. Arvot ovat siis epsg.io:sta kultakin koordinaatistolta (projected bounds -arvo), tässä esimerkkinä GK23FIN: https://epsg.io/3877-4326

Reply:
Also note to self, nämä voisi muuttaa `BoundingBox`:eiksi kokonaan erillisen tyypin sijasta